### ISSUE TITLE
Trophy alt-text

### ISSUE THREAD

**Issue by sprx97** [Spartan97/OldTimeHockey#45]
Show which div/league/year a trophy was earned in in the leaderboard view.

**Comment by sprx97** [comment]
Added hover text. Still not alt text when failed to load, but probably not adding that.
